daemon/emer-daemon.c

```diff
@@ -1933,5 +2010,27 @@ emer_daemon_start_aggregate_timer (EmerDaemon          *self,
 
   g_hash_table_add (priv->aggregate_timers, g_steal_pointer (&timer_impl));
 
+  /* Monitor sender and shut down all timers from it when it vanishes */
+  sender_data = g_hash_table_lookup (priv->monitored_senders, sender_name);
```

Comment:
Drive-by thought: does GDBus do the same thing internally, in which case there is no advantage to coalescing senders together here?

Reply:
Looking at https://gitlab.gnome.org/GNOME/glib/-/blob/main/gio/gdbusnamewatching.c, I don't see this happening. But, I think it's actually good to do that ourselves - it's easy to do, literally involves 4 lines of code.